e2e/tests-dfx/assetscanister.bash

```diff
@@ -726,3 +726,52 @@ CHERRIES" "$stdout"
   "ignore": false
 }'
 }
+
+@test "asset configuration via .ic-assets.json5 - get and set asset properties" {
+    install_asset assetscanister
+
+    dfx_start
+
+    mkdir src/e2e_project_frontend/assets/somedir
+    touch src/e2e_project_frontend/assets/somedir/upload-me.txt
+    echo '[
+      {
+        "match": "**/*",
+        "cache": { "max_age": 2000 },
+        "headers": { "x-key": "x-value" }
+      }
+    ]' > src/e2e_project_frontend/assets/.ic-assets.json5
+
+    dfx deploy
+    ID=$(dfx canister id e2e_project_frontend)
+    PORT=$(get_webserver_port)
+
+    assert_command dfx canister --network "http://localhost:$PORT" call "$ID" get_asset_properties '("/somedir/upload-me.txt")'
```

Comment:
What is the rationale for passing the network by port here?  If not passing it, or even passing it by name, and possibly passing the canister by name, will the output `record` contain the field names rather than numbers? (`headers` rather than `1_661_489_734`)
```suggestion
    assert_command dfx canister call e2e_project_frontend get_asset_properties '("/somedir/upload-me.txt")'
```

Reply:
the rationale: 🤦 

so, I believe I added `--network ....` because it didn't work otherwise -- not sure why it works now, perhaps someone fixed the default port in `agent-rs`?

after removing the --network flag I can see field names listed correctly in the response (as opposed to numbers "`1_661_489_734`") - why? what's the reason it makes such a difference?

also gonna change the call by canister name to call by `ID`

Reply:
The reason it makes a difference is dfx isn't finding a candid file when the network is passed by URL.